### PR TITLE
Fix IE8 tests

### PR DESCRIPTION
**Pull request by heff**
- Removed incompatible module
- Added ie8 shim to qunit index.html
- Fixed extends() usage

### CODE REVIEW

**Comment by gkatsev** [comment]
`extends` was already fixed here: https://github.com/videojs/video.js/pull/2390
I just hadn't been able to pull it in yet, trying to wrestle with text tracks.

File: test/unit/tech/tech.test.js [hunk 1]
```diff
@@ -103,7 +104,7 @@ test('should add the source handler interface to a tech', function(){
   var sourceB = { src: 'no-support', type: 'no-support' };
 
   // Define a new tech class
-  var MyTech = Tech.extend();
+  var MyTech = extendsFn(Tech);
```
Comment (dmlap) [inline test/unit/tech/tech.test.js]: Just curious, why the `Fn` suffixing?

**Comment by dmlap** [comment]
modulo Gary's comment about the other PR for `extends`, lgtm

File: test/unit/tech/tech.test.js (hunk 1 above)
Comment (heff) [inline test/unit/tech/tech.test.js]: because `extends` is reserved. `class Blah extends Fblah`

**Comment by heff** [comment]
Pulled in #2390 and rebased this one. Merging in.

**Pull request merged**